python/lsst/afw/image/image.cc

```diff
@@ -224,22 +296,35 @@ PYBIND11_PLUGIN(_image) {
         .value("LOCAL", ImageOrigin::LOCAL)
         .export_values();
 
+    declareImageBase<int>(mod, "I");
     declareImageBase<float>(mod, "F");
     declareImageBase<double>(mod, "D");
-    declareImageBase<int>(mod, "I");
```

Comment:
Hmm, I always just used I, F, D, U, L. For this case it really shouldn't matter. But for overloads it might.